docs/products/mysql/get-started.md

```diff
@@ -3,36 +3,380 @@ title: Get started with Aiven for MySQL®
 sidebar_label: Get started
 ---
 
-Start using Aiven for MySQL® by creating and configuring a service.
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import ConsoleLabel from "@site/src/components/ConsoleIcons"
+
+Start using Aiven for MySQL® by creating a service, connecting to it, and loading sample data.
+
+## Prerequisites
+
+- Access to the [Aiven Console](https://console.aiven.io)
+- [MySQL CLI client](https://dev.mysql.com/doc/refman/8.0/en/mysql.html)
+  installed
+- [Terraform installed](https://developer.hashicorp.com/terraform/install) if you prefer
+  to get started using code
 
 ## Create a service
 
-[Create an Aiven for PostgreSQL® service](/docs/platform/howto/create_new_service) in the
+<Tabs groupId="group1">
+<TabItem value="1" label="Console" default>
+[Create an Aiven for MySQL® service](/docs/platform/howto/create_new_service)  in the
 [Aiven Console](https://console.aiven.io).
+</TabItem>
+<TabItem value="2" label="Terraform">
+
+1. [Create an authentication token](/docs/platform/howto/create_authentication_token).
+1. Store the authentication token in an environment variable:
+
+   ```bash
+   export TF_VAR_aiven_api_token=YOUR_AIVEN_API_TOKEN
+   ```
+
+1. Create the following Terraform files:
+
+   - ``provider.tf``, where you specify the version in the ``required_providers`` block
+
+      ```hcl
+      terraform {
+        required_providers {
+          aiven = {
+            source  = "aiven/aiven"
+            version = ">=4.0.0, < 5.0.0"
+          }
+        }
+      }
+
+      provider "aiven" {
+        api_token = var.aiven_api_token
+      }
+      ```
+
+   - ``mysql.tf``, where you include the ``aiven_mysql`` resource
+
+      ```hcl
+      resource "aiven_mysql" "mysql" {
+        project      = var.project_name
+        service_name = var.service_name
+        cloud_name   = var.cloud_name
+        plan         = var.service_plan
+      }
+
+      output "mysql_service_host" {
+        value = aiven_mysql.mysql.service_host
+      }
+
+      output "mysql_service_port" {
+        value = aiven_mysql.mysql.service_port
+      }
+
+      output "mysql_service_username" {
+        value = aiven_mysql.mysql.service_username
+      }
+
+      output "mysql_service_password" {
+        value     = aiven_mysql.mysql.service_password
+        sensitive = true
+      }
+      ```
+
+   - ``variables.tf``, where you declare the API token and project name variables
```

Comment:
Since this is not only about those 2 variables, I think something more general would be better, e.g. ", where you declare your project variables."